sense/downstream_tasks/gesture_recognition/__init__.py

```diff
@@ -32,3 +32,33 @@
 }
 
 INT2LAB = {value: key for key, value in LAB2INT.items()}
+
+LAB_THRESHOLDS = {
+    "\"Drinking\" gesture": .2,
+    "\"Sleeping\" gesture": .7,
+    "Calling someone closer": .5,
+    "Covering ears": .5,
+    "Covering eyes": .5,
+    "Dabbing": .5,
+    "Facepalming": .5,
+    "Nodding": .5,
+    "Pointing left": .3,
+    "Pointing right": .3,
+    "Pointing to the camera": .5,
+    "Putting finger to mouth": .5,
+    "Rolling hand": .5,
+    "Scratching": .5,
+    "Shaking head": .5,
+    "Showing the middle finger": .3,
```

Comment:
I wasn't able to activate this at all :shrug: Does it work for you?

Reply:
It worked for me but I had to lower it to .3, should we lower it even further?

Reply:
I wouldn't want to have false positives on this one. If it worked for you, let's leave it there.